src/arch/MovieTexture/MovieTexture_Generic.cpp

```diff
@@ -10,7 +10,9 @@
 #include "Sprite.h"
 
 #include <cmath>
+#include <chrono>
```

Comment:
see comment below

Reply:
Removed in this file. But see my comment about the 1000fps point as it's relevant.